src/examples/Examples.js

```diff
@@ -96,48 +145,66 @@ var Examples = React.createClass({
           {blogPostCard}
           {featuredPostCard}
           {featuredPostCard}
-          {blogPostCard}
+          {socialFeedCard3}
         </Tiles>
       </Box>
     );
   },
 
-  _renderContentCards: function () {
+  _renderCards: function () {
     return (
       <Box pad={{horizontal: 'large'}}>
         <Tiles size="large" colorIndex="light-2" justify="center">
-          <ContentCard
+          <Card
+            onClick={this._onClickCard.bind(this, window.location.pathname + '#')}
             direction="column"
             thumbnail="/docs/img/Case_Study_image.png"
-            overline="Featured Post"
-            heading="Protect Your Digital Enterprise ipsum Learn More lorem dolores aeat"
-            description="It’s not an either/or world. It’s about finding the right platform for each app, workload and service. Learn how hybrid infrastructure can help you achieve cloud agility with traditional IT predictability. It’s not an either/or world. It’s about finding the right platform for each app, workload and service. Learn how hybrid infrastructure can help you achieve cloud agility with traditional IT predictability. It’s not an either/or world. It’s about finding the right platform for each app, workload and service. Learn how hybrid infrastructure can help you achieve cloud agility with traditional IT predictability. "
-            link="http://grommet.io"
+            label="Featured Post"
+            heading="Protect Your Digital Enterprise ipsum dolores aeat"
+            description={`It’s not an either/or world. It’s about finding the
```

Comment:
Can description be a string rather than an attribute expression?

```javascript 
description="It’s not an either/or world. It’s about finding the..."
```